ws-nextjs-app/pages/[service]/send/[id]/FormContext/utils/formTranslations/es.ts

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
I am wondering if it might be best to use the service files for this, instead of creating a whole bunch of new files for each language? We do have the service, so we could always look up the translations from the service context?